app/requests.js

```diff
@@ -15,12 +16,16 @@ module.exports = {
     } else {
       request({
           url: opts.url,
-          rejectUnauthorized: false,
+          rejectUnauthorized: false,    // Don't validate SSL certs
           headers: opts.headers || {},
           json: true
         },
         function (error, response, body) {
-          callback(error, body);
+          if (response.statusCode === 200) {
+            callback(error, body);
+          } else {
+            callback('HTTP Reponse: '+response.statusCode+' '+http.STATUS_CODES[response.statusCode]);
```

Comment:
We have to be careful here. Request usually returns an error object if understand [the code](https://github.com/request/request/blob/a6741d415aba31cd01e9c4544c96f84ea6ed11e3/request.js#L209) correctly.

Please ensure, that it works in exactly the same way and _only append_ the informational error message. Then I'm fine with the change.